content/tokio/glossary.md

```diff
@@ -61,16 +61,15 @@ combined part is said to be "in the same task".
 Multiple tasks are required for parallelism, but it is possible to concurrently
 do multiple things on one task using tools such as `join!`.
 
-[`tokio::spawn`]: https://docs.rs/tokio/1/tokio/fn.spawn.html
 [`Runtime::block_on`]: https://docs.rs/tokio/1/tokio/runtime/struct.Runtime.html#method.block_on
 [`join!`]: https://docs.rs/tokio/1/tokio/macro.join.html
+[`tokio::spawn`]: https://docs.rs/tokio/1/tokio/fn.spawn.html
 
 ## Spawning
 
-Spawning is when the `tokio::spawn` function is used to create a new task. It
+Spawning is when the [`tokio::spawn`] function is used to create a new task. It
 can also refer to creating new thread with [`std::thread::spawn`].
 
-[`tokio::spawn`]: https://docs.rs/tokio/1/tokio/fn.spawn.html
```

Comment:
Does this link still work now that the definition is above? I'd prefer to keep the second definition rather than the first so that the destination is after the use.

Reply:
it does still work, but your suggestion makes sense regardless. Thanks.